tasks/install-enketo.yml

```diff
@@ -143,6 +143,19 @@
   notify:
     - "reload pm2-{{ enketo_user }}"
 
+- name: Replace pm2 resurrect with pm2 start
+  lineinfile:
+    regexp: "^ExecStart"
+    line: 'ExecStart={{ enketo_pm2_path }} start app.js -n {{ enketo_user }} -i 0'
+    dest: "/etc/systemd/system/pm2-enketo.service"
+  when: enketo_init_system == "systemd"
+
+- name: Add WorkingDirectory=/home/enketo/app
+  lineinfile:
+    path: /etc/systemd/system/pm2-enketo.service
+    insertbefore: '^ExecStart'
+    line: 'WorkingDirectory=/home/enketo/app'
```

Comment:
@FrankApiyo I think we should replace `/home/enketo/app` with an existing variable.
Also, I think we should have both tasks in a single [ansible block](https://docs.ansible.com/ansible/latest/user_guide/playbooks_blocks.html). The the condition of the block would then be what we have in like 151

Reply:
amended the commit to address this